<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1a1a1a;
color: #ffffff;</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>600</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #242424;
font: 15px &quot;Lato Black&quot;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2d2d2d;
border:none;</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2d2d2d;
border:none;</string>
    </property>
    <property name="text">
     <string>APPLY</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="title_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato Black</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#242424</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffbe00;
font: 23px &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>FAVORITE</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>600</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	font: 15px &quot;Lato&quot;
}
QFrame{
	background:transparent;
	border: 3px solid #000000;
}
QCheckBox{
	background: none;
	color: #7f7f7f;
	font: bold 15px &quot;Lato&quot;
}
QCheckBox:Checked{
	color: #ffbe00;
}
QRadioButton{
	background: none;
	color: #7f7f7f;
	font: bold 15px &quot;Lato&quot;
}
QRadioButton:Checked{
	color: #ffbe00;
}
QLabel{
	border: none;
	font: 18px &quot;Lato&quot;;
	color: #dddddd;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QScrollArea" name="favorArea">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>360</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea{
	background: transparent;
	border: 3px solid #000000;
}
QScrollBar{
	width: 10px;
	background: #252525;
	border:none;
	margin: 0px 0px 0px 0px;
}
QScrollBar::handle{
	background: #1a1a1a;
}
QScrollBar::add-line, QScrollBar::sub-line{
	border:none;
	background: none;
}
QScrollBar::add-page, QScrollBar::sub-page{
	border:none;
	background: none;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="favor_list">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>344</width>
       <height>344</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="favor_search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background: #2d2d2d;
	border: none;
	color: #ffffff;
}</string>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="favor_ena">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Enabled</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="favor_dis">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Disabled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="favor_black">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Black List</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="favor_all">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Others</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
